src/routes/resources/[resourceContentId]/sidebar-content-store.ts

```diff
@@ -97,19 +97,8 @@ export function createSidebarContentStore(resourceContent: ResourceContent) {
                 snapshotOrVersion.isPublished ? ' (Published)' : ''
             }`;
         } else {
-            const isEnglish = resourceContent.language.iso6393Code.toLowerCase() === 'eng';
-
-            const isTranslatedFirstSnapshot =
-                isFirstSnapshot &&
-                snapshotOrVersion?.status === ResourceContentStatusDisplayEnum.AquiferizeEditorReview &&
-                !isEnglish;
-
-            const isEnglishFirstSnapshot = isFirstSnapshot && isEnglish;
-
-            if (isEnglishFirstSnapshot || isTranslatedFirstSnapshot) {
+            if (isFirstSnapshot) {
                 return `${formatDate(snapshotOrVersion.created)} - Source`;
-            } else if (isFirstSnapshot) {
-                return `${formatDate(snapshotOrVersion.created)} English Source`;
```

Comment:
See https://biblionexusworkspace.slack.com/archives/C05K5N7F44Q/p1739905461468839 for discussion on this change.